src/libraries/System.Text.Json/gen/Reflection/RoslynExtensions.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace System.Text.Json.Reflection
 {
-    internal static partial class RoslynExtensions
+    internal static class RoslynExtensions
```

Comment:
Should this stay as the same type and move to the same namespace `Microsoft.CodeAnalysis.Shared.Extensions`?

Reply:
If you would prefer having the same type\namespace, I'll change it. Not sure if the original intent was to keep these as STJ-specific.

Reply:
Okay to leave as-is. With both approaches, they are internal to JSON only.